public/employers/tesla.yml

```diff
@@ -77,10 +77,13 @@ citations:
       - source: Forbes
         link: https://www.forbes.com/sites/billroberson/2020/04/04/health-officials-say-ventilators-donated-by-tesla-are-wrong-type-and-not-powerful-enough/#26d87acb6264
         date: 2020-04-04T15:00:00-04:00
-  - summary: Elon Musk has announced a repurposing of Tesla's solar-panel factory in New York to produce ventilators to combat COVID-19. However, this plan has yet to come to fruition as of April 4.
+  - summary: Elon Musk announced a repurposing of Tesla's solar-panel factory in New York to produce ventilators to combat COVID-19. Nothing more was heard of this plan until April 7, when Tesla showed off its new ventilator prototypes. As of April 23, nothing further had been announced or revealed.
```

Comment:
The ventilators were delivered, but needed some modifications to get it working at full capacity. 
https://www.bloomberg.com/news/articles/2020-04-17/hospitals-made-use-of-modified-tesla-donated-breathing-machines

Reply:
Oh nice, I'll add this.